### PR TITLE
Add adopter China Mobile, e签宝，Trendyol，cestc，Siemens Tech

### PR DESCRIPTION
**Pull request by jefree-cat**
Signed-off-by: jefree-cat <jiahang_xu@cmbchina.com>
add new users

### CODE REVIEW

**Comment by jefree-cat** [comment]
@wonderflow

**Review by Somefive — APPROVED** [review]
Great!

File: ADOPTERS.md [hunk 1]
```diff
@@ -50,4 +50,9 @@ This page contains a list of organizations who are using KubeVela in production
...
-| [铭师堂](https://www.mistong.com/)                                                              | ![testing](https://img.shields.io/badge/-development%20&%20testing-green?style=flat) |                                                                                                                                                                                                                                                                                             |
\ No newline at end of file
+| [铭师堂](https://www.mistong.com/)                                                              | ![testing](https://img.shields.io/badge/-development%20&%20testing-green?style=flat) |                                                                                                                                                                                                                                                                                             |
+| China Mobile						                                                              | ![testing](https://img.shields.io/badge/-development%20&%20testing-green?style=flat) |
```
Comment (wonderflow) [inline ADOPTERS.md:54]: China mobile is in production now

Comment (jefree-cat) [reply ADOPTERS.md:54]: Cool

**Pull request merged**